Makefile

```diff
@@ -11,6 +11,6 @@ checkdeps:
 
 install: checkdeps
 	install -d /usr/share/tt-rss/www/plugins/ff_xmllint
-	install -m 644 ff_xmllint/init.php /usr/share/tt-rss/www/plugins/ff_xmllint/
+	install -m 644 init.php /usr/share/tt-rss/www/plugins/ff_xmllint/
```

Comment:
the Makefile is meant to run from the repo root, and in that case the `init.php` is very definitely in the `ff_xmllint` subdirectory.

For this change to work, you would need to move everything out of the subdirectory, which isn't in this PR. Did you miss committing or pushing a bit?

Reply:
I thought I also did that. Not sure why it is missing.

The goal was to allow install from git, which works best when everything is in the root folder.

Reply:
Maybe I forgot to push the change. Now, the move should be there.

Reply:
still isn't pushed